pkg/controller/issueapi/issue.go

```diff
@@ -156,5 +181,8 @@ func (ic *IssueAPI) getAuthorizationFromContext(r *http.Request) (*database.Auth
 		return nil, nil, fmt.Errorf("unable to identify authorized requestor")
 	}
 	return nil, user, nil
-
 }
+
+const smsTemplate = `Your exposure notifications verification code is %s. ` +
```

Comment:
this is fine for now, we'll probably need PHA customization here

and we'll need a URI to the app at some point.

Reply:
Yea I figured we'd iterate on it.